src/content/en/fundamentals/media/manipulating/cheatsheet.md

```diff
@@ -0,0 +1,372 @@
+project_path: /web/_project.yaml
+book_path: /web/fundamentals/_book.yaml
+description: A summary of commands used to convert a raw mov file to an encrypted full HD file for web playback.
+
+{# wf_updated_on: 2017-06-05 #}
+{# wf_published_on: 2017-06-05 #}
+
+# Media Manipulation Cheat Sheet  {: .page-title }
+
+{% include "web/_shared/contributors/josephmedley.html" %}
+
+This document shows in order commands needed to get from a raw mov file to
+encrypted assets packaged for DASH or HLS. For the sake of having a goal to
+illustrate, I'm converting my source file to a bitrate of 8Mbs at a resolution
+of 1080p (1920 x 1980). Adjust these values as your needs dictate.
```

Comment:
Worth linking to the d.g.c media guide on VP9 for representative bitrates etc for webm?